plugins/datalogger/src/dmm.cpp

```diff
@@ -66,14 +71,16 @@ QList<DmmDataMonitorModel *> DMM::getDmmMonitors(iio_context *ctx)
 						dmmInfo = m_hwmonDevices.value(static_cast<hwmon_chan_type>(type));
 					}
 					unitOfMeasurement = new UnitOfMeasurement(dmmInfo.name, dmmInfo.symbol);
-					dmmReadStrategy->setUmScale(dmmInfo.scale);
+					scale = scale * dmmInfo.scale;
```

Comment:
I think we need to keep these 2 separated if we want to override scale. The user should only override the hardware scale, not the UM scale.